### PR TITLE
fix(techdocs): copy to clipboard in not secure context

### PR DESCRIPTION
**Pull request by antoniomuso**
## Hey, I just made a Pull Request!

Hi, I also added the copy button to work in an insecure context, for example, in simple HTTP. 
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/techdocs/src/reader/transformers/copyToClipboard.tsx
Comment (vinzscam) [inline plugins/techdocs/src/reader/transformers/copyToClipboard.tsx:77]: you could reuse `useCopyToClipboard` (it's already used by other plugins), so that we don't introduce/maintain an ad-hoc method

Comment (antoniomuso) [reply plugins/techdocs/src/reader/transformers/copyToClipboard.tsx:77]: okay

**Review by agentbellnorm — APPROVED** [review]
Looks good to me! Let's wait for a 👍 from @vinzscam before we merge.

**Review by Rugvip — APPROVED** [review]
Great, thanks! 👍

**Pull request merged**
